src/Meshes/abstractMesh.ts

```diff
@@ -1099,14 +1099,43 @@ export class AbstractMesh extends TransformNode implements IDisposable, ICullabl
             return this._masterMesh.getBoundingInfo();
         }
 
-        if (!this._boundingInfo) {
-            // this._boundingInfo is being created here
+        if (!this._boundingInfo || this._boundingInfoIsDirty) {
+            this._boundingInfoIsDirty = false;
+            // this._boundingInfo is being created if undefined
             this._updateBoundingInfo();
         }
         // cannot be null.
         return this._boundingInfo!;
     }
 
+    /**
+     * Overwrite the current bounding info
+     * @param boundingInfo defines the new bounding info
+     * @returns the current mesh
+     */
+    public setBoundingInfo(boundingInfo: BoundingInfo): AbstractMesh {
```

Comment:
Not sure these should be public. They were public _ before right?

Reply:
yes but I use that change as an opportunity to open the API a bit